src/core/components/accordion/accordionItem/accordionItem.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
A general feedback regarding the unit tests. On each unit test, we should only test the functionalities of the related component, otherwise we'll need to update all the tests when something changes on one component. Also, we don't need to test the functionalities already provided by third party libraries. 
Given that most of these components are pretty simple without much custom logic, this mean that we can have the following tests:
- accordionItem: renders the children property
- accordionItemContent: renders the children property
- accordionHeader: renders the children property / renders a button to open/close the accordion content / renders the correct icon